src/background.ts

```diff
@@ -192,120 +130,110 @@ async function handleMessage(p: Runtime.Port, message: any) {
       } else if (message["type"] === "mapResource") {
         console.debug("Generating map query");
         query = await generateMapQuery(message);
-      }else if (message["type"] === "mapSearch") {
+      } else if (message["type"] === "mapSearch") {
         console.debug("Generating map query");
         query = await generateMapSearchQuery(message);
-      }else if (message["type"] === "chartResource"){
+      } else if (message["type"] === "chartResource") {
         query = await generateChartQuery(message);
-      }else{
-        query = generateLocalQuery(message);
+      } else {
+        query = await generateLocalQuery(message);
       }
       if (message["toRender"] === "full") {
-        let audio = new Audio(browser.runtime.getURL("progressBar/image_request_sent.mp3"));
+        let audio = new Audio(browser.runtime.getURL("progressBar/audio-files/image_request_sent.mp3"));
         audio.play();
-        await getAllStorageSyncData().then(async items => {
-          if(items["mcgillServer"]===true){
-            serverUrl = "https://image.a11y.mcgill.ca/";
-          }else{
-            if(items["inputUrl"]!== "" && items["customServer"]===true){
+        let items = await getAllStorageSyncData();
+        if (items["mcgillServer"] === true) {
+          serverUrl = "https://image.a11y.mcgill.ca/";
```

Comment:
This URL is repeated in multiple places. Would be best to set this as a constant variable somewhere. Unlikely this will change so low priority.